fishcore/ClientApp/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This file contains all of the packages used by the React portion of the application